konami/product/GCC44-JA.md

```diff
@@ -0,0 +1,28 @@
+# `GCC44-JA`: Beatmania IIDX 8th Style software upgrade kit
```

Comment:
It's probably better to merge all IIDX products into something like `products/iidx.md` for now, until it gets too unwieldy.

Reply:
I had that before, but I went for this approach because you don't have to know the game series to find additional, though optional, detailed information about the specific product. We simply denote and identify the product with the product ID. It's a more scalable structure and an overview should rather be provided by the `products.md` file instead.

I am fine reverting that also as I had it like you suggested before. However, I already felt like that text file was getting too crowded with details about each section deserving separate files.

Reply:
Yeah, but my argument would be here that you can use `products.md`'s list of gamecodes to find it. :P